.github/actions/setup-conda/action.yml

```diff
@@ -21,8 +27,10 @@ runs:
       uses: conda-incubator/setup-miniconda@v2.1.1
       with:
```

Comment:
Looks like this is missing the `use-only-tar-bz2: true` option

Reply:
Thanks!